pkgs/by-name/ms/mstflint/package.nix

```diff
@@ -27,12 +27,11 @@
 stdenv.mkDerivation rec {
   pname = "mstflint";
 
-  # if you update the version of this package, also update the input hash in mstflint_access!
-  version = "4.34.0-1";
+  version = "4.34.1-2";
 
   src = fetchurl {
```

Comment:
This should use `fetchFromGitHub` now that it only fetches the "normal" sources.